### PR TITLE
Refactor for gradle

### LINKED ISSUE #5: Refactor for Gradle
This will make development significantly easier, and more platform/configuration-independent

- [x] Refactor
- [x] Add submodules as appropriate
- [ ] Update documentation for usage, building, and development

### PR DESCRIPTION
**Pull request by qbowers**
Closes #5

### CODE REVIEW

**Comment by qbowers** [comment]
Also closes #1 !

File: app/src/main/java/ihm/IhmEasyGec.java [hunk 1]
```diff
@@ -1,1582 +1,1582 @@
...
-      public void actionPerformed(ActionEvent e) 
-      {
-        // Lancement du fichier
-        Runtime r = Runtime.getRuntime();
-        
-        try
-        {
-          // Lancement du fichier d'aide de l'application
-          String adresse = new File(".").getCanonicalPath().toString();
-          adresse = "cmd /c start \"\" \""  + easyGec.getFichier().substring(0, easyGec.getFichier().length()-4) + EasyGec.getLangages().getText("113", EasyGec.getLang()) + ".html" +"\"";
```
Comment (oomip) [inline app/src/main/java/ihm/IhmEasyGec.java:1010]: only works on Windows

**Review by oomip — APPROVED** [review]
Functions on Mac + Linux.

Missing output functionality outside of Windows -- see line 1010 on IhmEasyGec.java

**Comment by qbowers** [comment]
Made new issue for this #21

**Pull request merged**